features/all-positions/LiquidationActionDialog.tsx

```diff
@@ -88,7 +88,7 @@ const LiquidationActionDialog = (props: DialogProps) => {
   } = empState;
 
   const prettyBalance = (x: number) => {
-    const x_string = x.toFixed(4);
+    const x_string = x.toFixed(8);
```

Comment:
does this not mess up the scaling when `prettyBalance` is used?

Reply:
It does but for USDBTC we need more than 4 decimals....